projects/go-tester/src/app/components/off-canvas-test/off-canvas-test.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="go-container">
-  <go-off-canvas-header-actions>
+  <!-- <go-off-canvas-header-actions>
     <go-button>Close</go-button>
-  </go-off-canvas-header-actions>
+  </go-off-canvas-header-actions> -->
```

Comment:
If you rebase this PR with `dev` this should no longer be an issue.

Reply:
Sure. I will commit that code once again. Please check.